packages/core-components/report-testUtils.api.md

```diff
@@ -3,8 +3,31 @@
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
+// @public (undocumented)
+export type Breakpoint = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
```

Comment:
Strange, thus isn't exported by MUI but it is defined internally O_o

I'm thinking it might be best to inline these for now, doesn't seem useful to have in the external API

Reply:
They are exported from MUI, thanks for pointing that out